fem/lininteg.cpp

```diff
@@ -735,4 +735,61 @@ void DGElasticityDirichletLFIntegrator::AssembleRHSElementVect(
    }
 }
 
+//L(v) := tau * (f, grad q)
+void StabGradDomainLFIntegrator::AssembleRHSElementVect(const FiniteElement &el,
```

Comment:
Do the changes for `StabGradDomainLFIntegrator` need to be part of this PR? (I'm not opposed to adding new integrators, but maybe in a separate PR).

Reply:
No problem. We can add it in pumi-NS merge.